items/COLE_MAYOR_MONSTER.json

```diff
@@ -1,28 +1,33 @@
 {
   "itemid": "minecraft:skull",
-  "displayname": "§eCole (Mayor)",
-  "nbttag": "{HideFlags:254,SkullOwner:{Id:\"ae44b3b0-f0fc-4a25-8fa9-2dccabb34d10\",Properties:{textures:[0:{Value:\"ewogICJ0aW1lc3RhbXAiIDogMTU5Nzc4MDkwNjkxMiwKICAicHJvZmlsZUlkIiA6ICI0MWQzYWJjMmQ3NDk0MDBjOTA5MGQ1NDM0ZDAzODMxYiIsCiAgInByb2ZpbGVOYW1lIiA6ICJNZWdha2xvb24iLAogICJzaWduYXR1cmVSZXF1aXJlZCIgOiB0cnVlLAogICJ0ZXh0dXJlcyIgOiB7CiAgICAiU0tJTiIgOiB7CiAgICAgICJ1cmwiIDogImh0dHA6Ly90ZXh0dXJlcy5taW5lY3JhZnQubmV0L3RleHR1cmUvMTY0MjJkZTA4ODQ4OTUyZDFjYmVhZDY2YmJiYWQ2ZjA3MTkxYmRjYzk1MmYzZDEwMzZhZWIwYzIyOTM4ZjM5YiIKICAgIH0KICB9Cn0=\"}]}},display:{Lore:[0:\"§8Candidate\",1:\"\",2:\"§eMining XP Buff\",3:\"§7Get §3+60☯ Mining Wisdom §7on\",4:\"§7public islands.\",5:\"\",6:\"§eProspection\",7:\"§7Mining minions work §a25%\",8:\"§7faster.\",9:\"\",10:\"§eMining Fiesta\",11:\"§7Schedules two Mining Fiesta events during the year!\",12:\"§7Earn §d2x §7drops from mining and extra §eUnique Loot§7!\",13:\"§7Grants §3+75☯ Mining Wisdom§7 during the event.\",14:\"§8Only active on public islands.\"],Name:\"§eCole (Mayor)\"},ExtraAttributes:{id:\"COLE_MAYOR_MONSTER\"}}",
+  "displayname": "§dCole (Mayor)",
+  "nbttag": "{HideFlags:254,SkullOwner:{Id:\"ae44b3b0-f0fc-4a25-8fa9-2dccabb34d10\",Properties:{textures:[0:{Value:\"ewogICJ0aW1lc3RhbXAiIDogMTU5Nzc4MDkwNjkxMiwKICAicHJvZmlsZUlkIiA6ICI0MWQzYWJjMmQ3NDk0MDBjOTA5MGQ1NDM0ZDAzODMxYiIsCiAgInByb2ZpbGVOYW1lIiA6ICJNZWdha2xvb24iLAogICJzaWduYXR1cmVSZXF1aXJlZCIgOiB0cnVlLAogICJ0ZXh0dXJlcyIgOiB7CiAgICAiU0tJTiIgOiB7CiAgICAgICJ1cmwiIDogImh0dHA6Ly90ZXh0dXJlcy5taW5lY3JhZnQubmV0L3RleHR1cmUvMTY0MjJkZTA4ODQ4OTUyZDFjYmVhZDY2YmJiYWQ2ZjA3MTkxYmRjYzk1MmYzZDEwMzZhZWIwYzIyOTM4ZjM5YiIKICAgIH0KICB9Cn0\u003d\"}]}},display:{Lore:[0:\"§8Candidate\",1:\"\",2:\"§dProspection\",3:\"§7§7Mining minions work §a25% §7faster.\",4:\"\",5:\"§dMining XP Buff\",6:\"§7§7Get §3+60☯ Mining Wisdom §7on public\",7:\"§7§7islands.\",8:\"\",9:\"§dMining Fiesta\",10:\"§7§7§7Schedules 5 §6Mining Fiestas\",11:\"§7§6§7throughout the year! During these\",12:\"§7§7events gain §3+75☯ Mining Wisdom§7,\",13:\"§7§7collect §9Refined Mineral §7and §5Glossy\",14:\"§7§5Gemstone§7, and earn §d2x drops §7from\",15:\"§7§7mining!\",16:\"\",17:\"§dMolten Forge\",18:\"§7§7Decrease the time it takes to\",19:\"§7§7§6forge §7items by §d25%§7.\"],Name:\"§dCole (Mayor)\"},ExtraAttributes:{id:\"COLE_MAYOR_MONSTER\"}}",
   "damage": 3,
   "lore": [
     "§8Candidate",
     "",
-    "§eMining XP Buff",
```

Comment:
The reason this is §e is to differentiate easily between mayors with colors, this conflicts with foxy.

Reply:
reverted